unit-3-js-labs/counting-loops/index.js

```diff
@@ -0,0 +1,48 @@
+for (let i = 1; i<=10; i++) {
+    console.log (i)    
+    for (let j = 10; j >= 1; j--) {  
+        console.log(j)
+    }
+}
+
+
+let k = 1
+while (k <= 10) {
+    console.log(k)
+    let l = 10
+    while (l >= 1) {
+        console.log(l)
+        l--
+    }
+    k++
+}
+
+
+let m = 1
+do {
+    console.log(m)
+    let n = 10
+    do {
+        console.log(n)
+        n--
+    } while (n >= 1)
+    m++
+} while (m <= 10)
+
+
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+for (let p = 0; p < numbers.length; p++) {
+    const number = numbers[p]
+	console.log(number)
+}
```

Comment:
While this works, it can be simplified by writing a `for ... of` loop

```suggestion
const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
for (const number of numbers) {
    console.log(number)
} 
```